src/pages/applications.js

```diff
@@ -313,6 +314,21 @@ const emergingProjects = [
       {label: 'Website', url: 'https://www.loxilb.io/'},
     ],
   },
+  {
+    logoUrl: 'https://skywalking.apache.org/',
+    name: 'Apache SkyWalking',
+    logo: skywalkingLogo,
+    title: 'APM, Application Performance Monitoring System',
+    description: `Apache SkyWalking is an application performance monitor tool for distributed systems, 
+    especially designed for microservices, cloud native and container-based (Kubernetes) architectures.
+    SkyWalking Rover is an agent in the SkyWalking ecosystem, as a metrics collector and profiler 
+    powered by eBPF to diagnose CPU, I/O and L4/L7(TLS) network performance. Also, Rover provides add-on events for 
+    spans in the distributed tracing.`,
+    urls: [
+      {label: 'GitHub', url: 'https://github.com/apache/skywalking'},
```

Comment:
Do you ask for this repo? I prefer the main repo, as the rover agent could only work when connected with SkyWalking backend. 

https://skywalking.apache.org/blog/2022-07-05-pinpoint-service-mesh-critical-performance-impact-by-using-ebpf/#zipkin-with-100-sampling

Reply:
Ideally it should be https://github.com/apache/skywalking-rover given the point is to show-case eBPF projects and in the main repo it's not obvious at all. From a user PoV it should be fine given you have the link to official doc there anyway, so easy to figure out. Could you also rebase to latest master, there are otherwise merge conflicts? Thx